### ISSUE TITLE
FanControl is running in task manager, but can't access GUI

### ISSUE THREAD

**Issue by Rosandal** [Rem0o/FanControl.Releases#863]
I can see the process is running. If I try to end the process, I am not allowed. It's like it is not reacting to it.

I deleted and redownloaded, but no luck. Anything else I can do? Any logs I can share?

It worked fine before upgrading my PSU. I just find it weird, if that would have any effect.

EDIT: I'm very eager to get it working again. Amazing piece of software!

**Comment by Rem0o** [comment]
Is your PSU a corsair one with fan control within ICUE?

**Comment by Rem0o** [comment]
Open your json config file with fancontrol off:

![image](https://user-images.githubusercontent.com/14118956/172697640-d49152fd-6fd2-4aa2-9e50-a15b9f2485e5.png)

Scroll all the way down and put psu to false, then save.

**Comment by Rosandal** [comment]
I'm sure that would fix it. I'm not sure which file. Here is a screenshot of those I have atm.

[https://imgur.com/a/IBVCaHT](url)

**Comment by Rem0o** [comment]
There is no configs or trace that the program ever ran from that folder.... I thought you tried to start from an existing config.

**Comment by Rosandal** [comment]
> There is no configs or trace that the program ever ran from that folder.... I thought you tried to start from an existing config.

I did, but deleted everything to "reinstall" or "reextract", hoping that would fix the issue.
I have run the fancontrol.exe since extracting, and I can see it in task manager.

Sorry if that's confusing. So haven't been able to execute it, while on a fresh extract so to speak.

**Comment by Rosandal** [comment]
> There is no configs or trace that the program ever ran from that folder.... I thought you tried to start from an existing config.

A bit more information, as I didn't understand why it wouldn't work yesterday.

I swapped my PSU from an AX860 -> AX1600i. The link between the AX1600i and my PC is not connected. So it just delivers power. I suppose, this wouldn't conflict with FanControl?

To make troubleshooting easier and see if I could fix it, I just whiped my system completely and did a fresh Windows 11 install. I still have the same issue. It runs in task manager, but I can't open it.

I'm honestly lost at how to get this running again. I'll probably try again at a later release. 

If there is anything I can share or information I can give you, for your development purposes or benefit you in any way. Let me know. Till then, thanks for your quick replies and have a very nice day.

Ps. Amazing software you made. I haven't had any other made me do a clean install in the hopes of fixing it :P

**Comment by Rosandal** [comment]
Oh god, this has been resolved, and I'm not sure how this would cause an issue, however I will post it here, if it can help you in any way.

After swapping PSU, I did cable management. Every hard drive was detected. I have 3 drives, 1 NVME & 2x SSD's. The one SSD that I of course don't use, for some reason was causing the issue. I opened the case and unplugged it. Every single issue I had disappeared. Replugged it, and made sure the cable wasn't too tight. Now I have no issues and everything is up and running.

The best part is I get to use this great tool again.

Thanks for letting me now regarding Corsair PSU's and ICUE. I will likely be connecting it at some point and remember to change the config file.
